### ISSUE TITLE
Support for Google Gemma models?

### ISSUE THREAD

**Issue by trignomtry** [BrutalCoding/aub.ai#17]
In the official LLama.cpp repo, the Gemma Models are supported. Can this be supported? It just needs the latest version of Llama.cpp.

**Comment by BrutalCoding** [comment]
Yeah I was reading some comments from the authors of gemma.cpp on HN (HackerNews), and noticed some mentions of this being supported in llama.cpp too.

I haven't tried their model yet, so I will do some testing.

You're right, it's just a matter of updating llama.cpp, although I still manually boot up every OS to compile the binaries and test the example app on it.

I'll set time aside this week to at least update aub_ai on pub.dev with a recent version of llama.cpp included, along with anything else I can cramp in within the leftover hours.

Thanks,
Daniel
